sources/packages/backend/apps/db-migrations/src/sql/NotificationMessages/Create-notification-messages.sql

```diff
@@ -0,0 +1,34 @@
+CREATE TABLE IF NOT EXISTS sims.notification_messages (
+    id INT PRIMARY KEY,
+    description VARCHAR(1000) NOT NULL,
+    -- Audit columns
+    created_at TIMESTAMP WITH TIME ZONE NOT NULL DEFAULT now(),
+    updated_at TIMESTAMP WITH TIME ZONE NOT NULL DEFAULT now(),
+    creator INT NULL DEFAULT NULL REFERENCES users(id) ON DELETE
```

Comment:
Also missing sims prefix for users here and in the line 10.

Reply:
Added it.